.github/workflows/ci.yml

```diff
@@ -414,7 +414,7 @@ jobs:
           toolchain: nightly-2023-06-17
 
       - name: System dependencies
-        run: sudo apt-get update; sudo apt-get install -y libudev-dev libgbm-dev libxkbcommon-dev libegl1-mesa-dev libwayland-dev libinput-dev libdbus-1-dev libsystemd-dev libseat-dev
+        run: sudo apt-get update; sudo apt-get install -y libdrm-dev libudev-dev libgbm-dev libxkbcommon-dev libegl1-mesa-dev libwayland-dev libinput-dev libdbus-1-dev libsystemd-dev libseat-dev
```

Comment:
Do we need `libdrm-dev`, if we don't enable the feature by default?

Reply:
Well let's find out I don't have a way to test the ci locally so it's experiment away